services/kommander/0.6.0/dynamic-helmreleases/mtls-proxy/defaults/cm.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mtls-proxy-0.1.8-d2iq-defaults
```

Comment:
this might not work since we create multiple kustomizations per cluster that would deploy this same CM. so I either need to use a unique name here or move this back to the controller 🤔

Reply:
Can't we deploy this defaults cm as part of the kommander kustomization? Then it will be available to use when the controller creates the dynamic mtls proxy helmreleases.

Reply:
There should only be a single defaults configmap rather than one per deployment I think.

Reply:
yeah agreed, I’ll move the creation to the kommander kustomization and rename it so it’s only created once